KrampitzKreutzwiserSettlersOfCatan/src/dataFiles/roadData.txt

```diff
@@ -1,34 +1,34 @@
-691
+681
 377
 2
 0
 1
 
-691
+681
```

Comment:
Are you sure these values need to be changed? The old ones worked fine last night.

Reply:
oh give me a sec to commit another line i forgot to copy
the change that fixed needing to offset those numbers